docs/home/COMPONENTS.md

```diff
@@ -4,43 +4,43 @@
-| alertmanager              | 0.17.0  | https://github.com/prometheus/alertmanager            | [Apache License 2.0](https://github.com/prometheus/alertmanager/blob/master/LICENSE) |
-| ignite                    | 2.5.0   | https://github.com/apache/ignite                      | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
-| Apache2                   | 2.4.29  | https://httpd.apache.org/                             | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
-| Hashicorp Vault           | 1.4.0   | https://github.com/hashicorp/vault                    | [Mozilla Public License 2.0](https://github.com/hashicorp/vault/blob/master/LICENSE) |
-| Hashicorp Vault Helm Chart | 0.4.0   | https://github.com/hashicorp/vault-helm               | [Mozilla Public License 2.0](https://github.com/hashicorp/vault-helm/blob/master/LICENSE.md) |
-| Bitnami Node-Exporter Helm Chart | 1.1.2   | https://github.com/bitnami/charts                    | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
-| Filebeat Helm Chart       | 7.9.2   | https://github.com/elastic/helm-charts                   | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Component                  | Version  | Repo/Website                                          | License                                                           |
+| -------------------------- | -------- | ----------------------------------------------------- | ----------------------------------------------------------------- |
+| Kubernetes                 | 1.18.6   | https://github.com/kubernetes/kubernetes              | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Kubernetes Dashboard       | 2.0.3    | https://github.com/kubernetes/dashboard               | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Calico                     | 3.15.0   | https://github.com/projectcalico/calico               | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Flannel                    | 0.12.0   | https://github.com/coreos/flannel/                    | [Apache License](https://www.apache.org/licenses/LICENSE-1.0)     |
+| Canal                      | 3.15.0   | https://github.com/projectcalico/calico               | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Kafka                      | 2.6.0    | https://github.com/apache/kafka                       | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| Zookeeper                  | 3.5.8    | https://github.com/apache/zookeeper                   | [Apache License 2.0](https://www.apache.org/licenses/LICENSE-2.0) |
+| RabbitMQ                   | 3.8.9    | https://github.com/rabbitmq/rabbitmq-server           | [Mozilla Public License](https://www.mozilla.org/en-US/MPL/)      |
```

Comment:
Please check this file, since in v0.8 branch there is RabbitMQ 3.8.3

Reply:
@to-bar it was changed by me as I expected Rabbitmq backporting (PR already created) for 0.8.1 release. I'll check versions once again.

Reply:
I would revert this file to v0.8 branch version, then just update Docker related entry since this file contains also `Epicli binary dependencies` in versions from develop branch.

When I was working on upgrading Docker in develop branch I've used separate commits for easier back-porting. See [commits](https://github.com/epiphany-platform/epiphany/pull/1885/commits).